test/perf/kernel/go_benchmark.jl

```diff
@@ -91,261 +91,261 @@ POS(board::Board, i::Int, j::Int) = (j - 1) * board.size + i
 IJ(board::Board, pos::Int) = (1 + mod((pos - 1), board.size), 1 + fld(pos - 1, board.size))
 
 function set_komi(board::Board, komi::Float64)
-  board.komi = komi
+    board.komi = komi
 end
 
 function set_random_seed(board::Board, seed::UInt32)
-  xor_srand(board.rand, seed)
+    xor_srand(board.rand, seed)
 end
 
 function clear(board::Board)
-  board.board[:] = EMPTY
+    board.board[:] = EMPTY
 end
 
 is_pass_move(i::Int, j::Int) = i == 0 && j == 0
 
 function on_board(board::Board, i::Int, j::Int)
-  i >= 1 && i <= board.size && j >= 1 && j <= board.size
+    i >= 1 && i <= board.size && j >= 1 && j <= board.size
 end
 
 function legal_move(board::Board, i::Int, j::Int, color::Int)
-  other = other_color(color)
+    other = other_color(color)
 
-  # Pass is always legal.
-  if is_pass_move(i, j)
-    return true
-  end
+    # Pass is always legal.
+    if is_pass_move(i, j)
+        return true
+    end
 
-  # Already occupied.
-  if board[i, j] != EMPTY
-    return false
-  end
+    # Already occupied.
+    if board[i, j] != EMPTY
+        return false
+    end
 
-  # Illegal ko recapture. It is not illegal to fill the ko so we must
-  # check the color of at least one neighbor.
-  if i == board.ko_i && j == board.ko_j && ((on_board(board, i - 1, j) && board[i - 1, j] == other) || (on_board(board, i + 1, j) && board[i + 1, j] == other))
-    return false
-  end
+    # Illegal ko recapture. It is not illegal to fill the ko so we must
+    # check the color of at least one neighbor.
+    if i == board.ko_i && j == board.ko_j && ((on_board(board, i - 1, j) && board[i - 1, j] == other) || (on_board(board, i + 1, j) && board[i + 1, j] == other))
```

Comment:
Really long line here, might be nice to break it up. Maybe a line break after the last `||`?